### PR TITLE
Masking Ip address to first 2 octets

### PR DESCRIPTION
**Pull request by xiangtin**
# Summary
1. Masking of IP address to first 2 octets happens after the pivot value has been set
Bug link: https://dev.azure.com/microsoft/OS/_workitems/edit/52583280/

2. Fix typos
Bug link: https://dev.azure.com/microsoft/OS/_workitems/edit/52967334/

# Validation
https://ms.portal.azure.com/?feature.workbookGalleryBranch=xixiao%2Fmaskipaddress#view/Microsoft_Azure_WVD/WvdManagerMenuBlade/~/insights

image

### CODE REVIEW

**Review by mike-mo — CHANGES_REQUESTED** [review]
Please increase the version number by 0.0.1 for bug fixes

**Comment by xiangtin** [comment]
> Please increase the version number by 0.0.1 for bug fixes

Updated the version number to 4.1.1, since planning on merge #2755  first.

File: Workbooks/Windows Virtual Desktop/AtScale/Connection Reliability/Connection Reliability.workbook [hunk 1]
```diff
@@ -478,7 +478,7 @@
...
...ype_real, iif(\"ClientIPAddress\" == \"{DisconnectPivot}\", ClientIPAddress_real, iff(\"SessionHostIPAddress\" == \"{DisconnectPivot}\", SessionHostIPAddress_real, Pivot)))\r\n| join kind=leftouter\r\n(\r\n    WVDErrors\r\n    | where TimeGenerated > WindowStart\r\n    | summarize FirstError = min(TimeGenerated) by CorrelationId, HasErrors=True\r\n) on CorrelationId\r\n| extend EndDate=coalesce(FirstError, EndDate), HasErrors=coalesce(HasErrors, False) // If it has errors, take time of first error as endtime\r\n| extend ConnectionEndedHere=EndDate between(Probe..(WindowSize+Probe))\r\n| where (StartDate <= Probe and Probe <= EndDate)\r\n| extend ConnectionResult=iif(HasErrors and ConnectionEndedHere, \"ErrorConnection\", \"SuccessConnection\")\r\n| summarize Connections=dcountif(CorrelationId, ConnectionResult == \"SuccessConnection\"), Disconnected=dcountif(CorrelationId, ConnectionResult == \"ErrorConnection\") by Pivot, PivotColumn=\"{DisconnectPivot}\",  PivotValue\r\n| sort by Disconnected desc\r\n| extend Drop=100.0*Disconnected/(Connections+Disconnected), Total=Connections+Disconnected\r\n| extend Title = strcat(\"Filter for \", Pivot, \", details of time slice for \")\r\n",
```
Comment (mike-mo) [inline Workbooks/Windows Virtual Desktop/AtScale/Connection Reliability/Connection Reliability.workbook:481]: nit: the pattern to name columns "_real" could be confused with the schema disambiguation that Kusto uses for colliding column names with different types. I recommend "ClientTypeFull", "ClientIPAddressFull", "SessionHostIPAddressFull" instead (or *Unmasked for the IPs, etc.)

Comment (xiangtin) [reply Workbooks/Windows Virtual Desktop/AtScale/Connection Reliability/Connection Reliability.workbook]: Updated the variable name.

File: Workbooks/Windows Virtual Desktop/AtScale/Connection Reliability/Connection Reliability.workbook
Comment (mike-mo) [inline Workbooks/Windows Virtual Desktop/AtScale/Connection Reliability/Connection Reliability.workbook:1]: What if an IPv6 ever appears in this field? Is there a way to future-proof?

**Review by mike-mo — APPROVED** [review]

**Pull request merged**